graviti/manager/commit.py

```diff
@@ -162,7 +163,17 @@ def _getter(offset: int, limit: int) -> Dict[str, Any]:
                 limit=limit,
             )
 
-        total_count = _getter(0, 1)["total_count"]
+        count_criteria = deepcopy(criteria)
```

Comment:
why we need `deepcopy` here?

Reply:
To prevent the `criteria` from changing.